src/components/SpellingVariantInput.vue

```diff
@@ -39,7 +43,8 @@ const onSearchInput = ( inputValue: string ) => {
 	}
 
 	menuItems.value = wbLexemeTermLanguages.filter(
-		( lang ) => lang.label.startsWith( lowerCaseInputValue ) );
+		( lang ) => ( new RegExp( `\\b${inputValue}`, 'i' ) ).test( lang.label ),
```

Comment:
I think we should escape the `inputValue` here; thankfully, lodash has an [escapeRegExp](https://lodash.com/docs/4.17.15#escapeRegExp) function (found via [StackOverflow](https://stackoverflow.com/q/3446170/1420237)).

Reply:
It's not like they would gain anything from messing with that regex, but it doesn't hurt either

Reply:
Done.